values/gitea/gitea.gotmpl

```diff
@@ -0,0 +1,51 @@
+{{- $v := .Environment.Values }}
+{{- $o := $v.oidc }}
+{{- $g := $v.charts | get "gitea" dict }}
+{{- $k := $v.charts | get "keycloak" dict }}
+{{- $hasKeycloak := $k | get "enabled" true }}
+{{- $realm := $k | get "realm" "master" }}
+{{- $keycloakIssuer := printf "https://keycloak.%s/realms/%s" $v.cluster.domain $realm }}
+
+{{- $giteadomain := printf "gitea.%s" $v.cluster.domain}}
+{{- $addIgnorePolicyAnnotation := readFile "../../helmfile.d/utils/addIgnorePolicyAnnotation.gotmpl" }}
+
+nameOverride: gitea
+fullnameOverride: gitea
+# clusterDomain: {{ $v.cluster.domain }}
+
+gitea:
+  podAnnotations: 
+    {{-  tpl $addIgnorePolicyAnnotation (dict "policies" (list "psp-allowed-users"))  | nindent 4 }}
```

Comment:
My preference is to make explicit annotations. It would be good to establish some standard. @Morriz ?

Reply:
I mean I can add explicit annotations, but both the init container & normal container need root access. Hopefully with Gitea 1.14 a rootless image will be released, and we could upgrade this helm chart to use the proper security context and not need this annotation

Reply:
I understand. Please add `Hot issue` so we remember about it.

Reply:
I agree, we want to just set the annotations directly...lets do that from now on and refactor all the other tpl calls progressively (or in one go)